src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -141,50 +141,46 @@ internal fun ComponentIdentifier.resolvedVersion(): String? = when (this) {
  * Given [Configuration.getDependencies], return this dependency set as a set of identifiers, per
  * [ComponentIdentifier.toIdentifier].
  */
-internal fun DependencySet.toIdentifiers(
-  metadataSink: MutableMap<String, Boolean> = mutableMapOf()
-): Set<String> = mapNotNullToSet {
-  it.toIdentifier(metadataSink)
+internal fun DependencySet.toIdentifiers(): Set<Pair<String, Boolean>> = mapNotNullToSet {
```

Comment:
The `Pair<String, Boolean>` now stores the information if the dependency is supported or should be ignored for now.

In the future that could then be replaced by a Type that holds more information (metadata of the dependency). For example, the Jar the dependency will resolve to (see comment above).